internal/proxy/proxy.go

```diff
@@ -79,12 +83,16 @@ type Config struct {
 	// Postgres-only.
 	IAMAuthN bool
 
+	// PrivateIP enables connections via the database server's private IP address
+	// for all instances.
+	PrivateIP bool
+
 	// Instances are configuration for individual instances. Instance
 	// configuration takes precedence over global configuration.
 	Instances []InstanceConnConfig
 
 	// Dialer specifies the dialer to use when connecting to Cloud SQL
-	// instances.
+	// instances. Used by serverless.
```

Comment:
Let's leave this out for now. We need a separate commit explaining that the "public" interface is actually subject to change and is for internal use only, e.g., "Use at your own risk".